src/components/auth/LoginForm.tsx

```diff
@@ -0,0 +1,138 @@
+import { Formik } from "formik";
+import { useState } from "react";
+import { StyleSheet, View } from "react-native";
+import { Button, IconButton, Input, Link, Text, Title } from "@/components/common";
+import { ArrowRight, VisibilityOff, VisibilityOn } from "@/assets/icons";
+import { Colors, Fonts } from "@/styles/theme";
+import { useLogin } from "@/services/login";
+import { Notification } from ".";
+import { dictionaryMessage } from "@/constants/validation";
+
+export function LoginForm() {
+  const [hidePassword, setHidePassword] = useState(true);
```

Comment:
we have a pattern with our naming for boolean variables, we always have them in the format of a question. The example here would be something like isPasswordHidden